services/provider/server/server.go

```diff
@@ -690,3 +705,34 @@ func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStat
 
 	return &pb.ReportStatusResponse{}, nil
 }
+
+// PeerBlockPool RPC call to send the bootstrap secret for the pool
+func (s *OCSProviderServer) PeerBlockPool(ctx context.Context, req *pb.PeerBlockPoolRequest) (*pb.PeerBlockPoolResponse, error) {
```

Comment:
is there any chance that two clients call this rpc referring same block pool? if yes, the idea is to just fail one client & make it retry?

Reply:
Yes, right